### ISSUE TITLE
Breaks incoming HTML

### ISSUE THREAD

**Issue by clacke** [friendica/friendica#6030]
### Expected behavior

HTML coming in from AP and/or OStatus is just presented as-is, modulo any necessary sanitization.

### Actual behavior

HTML coming from AP and/or OStatus can be valid on native nodes, but become garbled when viewed on friendi.ca.

### Steps to reproduce the problem

Look at https://mastodon.social/@bjoern/100736446562569324:

```
[ . . . ] <a href="https://web.archive.org/web/20021217003716/http://www.opensource.org/advocacy/faq.html" rel="nofollow noopener" target="_blank"><span class="invisible">https://</span><span class="ellipsis">web.archive.org/web/2002121700</span><span class="invisible">3716/http://www.opensource.org/advocacy/faq.html</span></a> Also interesting in this context: <a href="https://lists.debian.org/debian-devel/1999/02/msg01641.html" rel="nofollow noopener" target="_blank"><span class="invisible">https://</span><span class="ellipsis">lists.debian.org/debian-devel/</span><span class="invisible">1999/02/msg01641.html</span></a></p><p>Cc <span class="h-card"><a href="https://mastodon.ar.al/@aral" class="u-url mention">@<span>aral</span></a></span></p>
```

This seems to be the original HTML sent over the wire, because it agrees with e.g. https://loadaverage.org/conversation/11079888#notice-13987581 .

Now granted, this has a lot of elements in there for various purposes, but there's nothing fishy -- neither nefarious nor ill-formed. Rendered, it looks something like (when viewed with Mastodon CSS, the URLs are shown shortened with trailing ellipses):

> [ . . . ] https://web.archive.org/web/20021217003716/http://www.opensource.org/advocacy/faq.html Also interesting in this context: https://lists.debian.org/debian-devel/1999/02/msg01641.html<p>Cc [@aral](https://mastodon.ar.al/@aral)</p>

----

Now look at https://libranet.de/display/52acf9cddb53e444ca9e79a10778201d004e768f :

```
[ . . . ] <a href="https://web.archive.org/web/20021217003716/<a%20href=" http:="" target="_blank">http://www.opensource.org/advocacy/faq.html</a>" target="_blank"&gt;https://web.archive.org/web/20021217003716/<a href="http://www.opensource.org/advocacy/faq.html" target="_blank">http://www.opensource.org/advocacy/faq.html</a> Also interesting in this context: <a href="https://lists.debian.org/debian-devel/1999/02/msg01641.html" target="_blank">https://lists.debian.org/debian-devel/1999/02/msg01641.html</a><br><br>Cc @<a href="https://mastodon.ar.al/@aral" class="userinfo mention" title="aral">aral</a><br>[attachment type='link' url='https://web.archive.org/web/20021217003716/<a href="http://www.opensource.org/advocacy/faq.html" target="_blank">http://www.opensource.org/advocacy/faq.html</a>' title='Open Source Initiative OSI -  FAQ:Advocacy']The Open Source Home Page[/attachment]
```

This renders something like:

> http://www.opensource.org/advocacy/faq.html" target="_blank">https:[]()//web.archive.org/web/20021217003716/http://www.opensource.org/advocacy/faq.html Also interesting in this context: https://lists.debian.org/debian-devel/1999/02/msg01641.html<p>Cc @aral [attachment type='link' url='https://web.archive.org/web/20021217003716/http://www.opensource.org/advocacy/faq.html' title='Open Source Initiative OSI - FAQ:Advocacy']The Open Source Home Page[/attachment]

### Friendica version you encountered the problem

`This is Friendica, version 2018.12-dev that is running at the web location https://libranet.de. The database version is 1289, the post update version is 1281.`

**Comment by annando** [comment]
Yes, Friendica does always convert incoming HTML to BBCode and then the exports are translating them to the different formats.

**Comment by MrPetovan** [comment]
Related to #4453 and #4451 and I should get back to #4488 to fix it.

**Comment by MrPetovan** [comment]
This particular issue (URLs inside other URLs) should be fixed by #6835
